group/group.go

```diff
@@ -30,6 +30,11 @@ type Group interface {
 	RandomScalar(io.Reader) Scalar
 	HashToElement(data, dst []byte) Element
 	HashToScalar(data, dst []byte) Scalar
+	Length() struct {
```

Comment:
Can we not do what Go does for curves, and make this a more generic "GroupParams" function?

Reply:
I just see your previous comment, will do some changes accrodingly. I like the idea of `GroupParams` structure

Reply:
Yet another try, pls take a look.